mgolubenkov/02_HTML/my_CALENDAR.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML  4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
+<!-- copy to http://htmlbook.ru/faq/kak-sozdat-tablitsu-v-html5-i-ukazat-ee-parametry-cherez-->
+<html>
+ 	<head>
+  		<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+  		<title>Тег TABLE</title>
+ 	</head>
+ 	<body>
+  		<table frame="значение">...</table>
+		<!-- Значения
+			void		Не отрисовывать границы.
+			border		Граница вокруг таблицы.
+			above		Граница по верхнему краю таблицы.
+			below		Граница снизу таблицы.
+			hsides		Добавить только горизонтальные границы (сверху и снизу таблицы).
+			vsides		Рисовать только вертикальные границы (слева и справа от таблицы).
+			rhs			Граница только на правой стороне таблицы.
+			lhs			Граница только на левой стороне таблицы.-->
+  		<table border="1" align="center" width="14%">
+   		<caption><b>My manth MARCH</b></caption>
+   		<tr>
+    		<th colspan="6">Март</th>
+   		</tr>
+   		<tr>
+    		<th><b>ПН</b></th>
```

Comment:
Could you please remove all b tags from all th tags, th tag already provide bold font